src/mongoose_scram.erl

```diff
@@ -107,9 +107,8 @@ mask(Key, Data) ->
 
 enabled(Host) ->
     case ejabberd_auth:get_opt(Host, password_format) of
```

Comment:
There is `ejabberd_auth:get_opt/3` which takes a `Default` value in case the parameter is not set. I think it'd be safer to use the `scram` value as the `Default`. What do you think?

Reply:
In the `mongoose_scram,` there is another call to `ejabberd_auth:get_opt/2`. I think it can also benefit from providing the default value as 3rd parameter.

Reply:
True, it might be more safe that way. Will have that changed.